### PR TITLE
Training set creator

### PR DESCRIPTION
**Pull request by njroussel**

### CODE REVIEW

**Comment by njroussel** [comment]
LGTM

File: sources/misc/config.cpp [hunk 1]
```diff
@@ -114,8 +114,8 @@ namespace tmd {
     int Config::features_comparator_sample_cols = 180;
     int Config::features_comparator_centers_file_rows = 2;
     int Config::features_comparator_centers_file_cols = 180;
-    std::string Config::features_comparator_centers_file_name =
-            "./res/cluster/clusterCenters.txt";
+    std::string Config::features_comparator_centers_file_name = "./res/cluster/cluster.kfc";
```
Comment (JBouron) [inline sources/misc/config.cpp:117]: What does "kfc" mean ?

Comment (njroussel) [inline sources/misc/config.cpp:117]: kmeans feature centers ;)

Comment (Rimbaud13) [inline sources/misc/config.cpp]: :)
(:
:)
(:
:)
(:
:)
(:
:)
(:
:)
(:
:)
(:
:)
(:
:)
(:
:)
(:
:)
(:

**Pull request merged**